tests/inputs/UserTest.php

```diff
@@ -0,0 +1,159 @@
+        $field = new Field(5, 5, 1);
+        $option = new Getopt();
+        $user = new User();
+        $input[] = "2,2,1";
+        $input[] = "exit";
+        $user->fillField($field, new Console(), $option);
+
+        $pattern = [
+            [0, 0, 0, 0, 0],
+            [0, 0, 0, 0, 0],
+            [0, 0, 1, 0, 0],
+            [0, 0, 0, 0, 0],
+            [0, 0, 0, 0, 0]
+        ];
+
+        $isEqual = $this->compare($field, $pattern);
+        $this->assertTrue($isEqual);
+    }
+
+    public function testInstantExit()
+    {
+        global $input;
+        $field = new Field(5, 5, 1);
+        $option = new Getopt();
+        $user = new User();
+        $input[] = "exit";
+        $user->fillField($field, new Console(), $option);
+
+        $pattern = [
+            [0, 0, 0, 0, 0],
+            [0, 0, 0, 0, 0],
+            [0, 0, 0, 0, 0],
+            [0, 0, 0, 0, 0],
+            [0, 0, 0, 0, 0]
+        ];
+
+        $isEqual = $this->compare($field, $pattern);
+        $this->assertTrue($isEqual);
+    }
+
+    public function testHitAndRun()
+    {
+        global $input;
+        $field = new Field(5, 5, 1);
+        $option = new Getopt();
+        $user = new User();
+        $input[] = "1,1";
+        $input[] = "1,1,0";
+        $input[] = "exit";
+        $user->fillField($field, new Console(), $option);
+
+        $pattern = [
+            [0, 0, 0, 0, 0],
+            [0, 0, 0, 0, 0],
+            [0, 0, 0, 0, 0],
+            [0, 0, 0, 0, 0],
+            [0, 0, 0, 0, 0]
+        ];
+
+        $isEqual = $this->compare($field, $pattern);
+        $this->assertTrue($isEqual);
+    }
+
+    public function testStateZeroStayZero()
+    {
+        global $input;
+        $field = new Field(5, 5, 1);
+        $option = new Getopt();
+        $user = new User();
+        $input[] = "1,1,0";
+        $input[] = "exit";
+        $user->fillField($field, new Console(), $option);
+
+        $pattern = [
+            [0, 0, 0, 0, 0],
+            [0, 0, 0, 0, 0],
+            [0, 0, 0, 0, 0],
+            [0, 0, 0, 0, 0],
+            [0, 0, 0, 0, 0]
+        ];
+
+        $isEqual = $this->compare($field, $pattern);
+        $this->assertTrue($isEqual);
+    }
+
+    public function testOutOfBounce()
+    {
+        global $input;
+        $field = new Field(5, 5, 1);
+        $option = new Getopt();
+        $user = new User();
+        $input[] = "-1,-1,1";
+        $input[] = "6,6,1";
+        $input[] = "exit";
+        $user->fillField($field, new Console(), $option);
+
+        $pattern = [
+            [0, 0, 0, 0, 0],
+            [0, 0, 0, 0, 0],
+            [0, 0, 0, 0, 0],
+            [0, 0, 0, 0, 0],
+            [0, 0, 0, 0, 0]
+        ];
+
+        $isEqual = $this->compare($field, $pattern);
+        $this->assertTrue($isEqual);
+    }
+
+    /**
+     * @param Field $field
+     * @param array $pattern
+     * @return bool
+     */
+    public function compare(Field $field, array $pattern): bool
```

Comment:
Please remove this method and use the combination `Field::createFromArray()` and `isEqualTo()` instead.